sdcm/sct_events.py

```diff
@@ -234,6 +234,27 @@ def __str__(self):
                 super(DisruptionEvent, self).__str__(), self)
 
 
+class FullScanEvent(SctEvent):
+    def __init__(self, type, loader_node, ks_cf, db_node_ip, result=None):
+        super(FullScanEvent, self).__init__()
+        self.type = type
+        self.loader_node = str(loader_node)
+        self.ks_cf = ks_cf
+        self.db_node_ip = db_node_ip
+        self.result = result
+        self.publish()
+
+    def __str__(self):
+        msg = "{0}: type={1.type} loader_node={1.loader_node} select_from={1.ks_cf} on db_node={1.db_node_ip}"
+        if self.result:
+            if self.result['stderr']:
+                self.severity = Severity.ERROR
```

Comment:
please move this check into __init__
__str__ shouldn't have side effects...

Reply:
done